conda_smithy/cli.py

```diff
@@ -902,6 +920,14 @@ class UpdateAnacondaToken(Subcommand):
         "update-binstar-token",
         "rotate-binstar-token",
     ]
+    ci_names = [
```

Comment:
Better to have this as immutable if it's going to be shared as a class attribute.